assets/js/entry.js

```diff
@@ -12,10 +13,24 @@ if (!Modernizr.dataset) {
 
 // Bring in foundation and custom libraries.
 require('foundation/js/foundation/foundation');
-var WOW = require('wow.js/src/wow.coffee');
+let WOW = require('wow.js/src/wow.coffee');
```

Comment:
For my own understanding, where are we deciding to use `let` vs. `var`? I was reading on ES6 and it looks like they are scoped differently, but it sounds like the recommendation is in general we should be using `let` going forward?